typescript/cli/src/utils/chains.ts

```diff
@@ -44,19 +43,14 @@ export async function runMultiChainSelectionStep(
       pageSize: 20,
     })) as string[];
     handleNewChain(chains);
-    if (chains?.length >= 2) return chains;
+    if (chains?.length >= 2 || !chainSelection) return chains;
```

Comment:
Nit: I personally like reading these conditions in the reverse, basically `if(badCondition) error`

and then just return chains if all none of the bad conditions triggered